rasterio/plot.py

```diff
@@ -67,6 +65,51 @@ def show(source, cmap='gray', with_bounds=True):
     else:  # pragma: no cover
         raise ImportError("matplotlib could not be imported")
 
+def get_plotting_extent(source):
```

Comment:
Since there's no function to set the plotting extent, let's call this `plotting_extent()`.